$OpenBSD: patch-common_argus_util_c,v 1.6 2021/02/25 21:31:28 naddy Exp $

First hunk: remove unused variable that clashes with one in raconvert

Index: common/argus_util.c
--- common/argus_util.c.orig
+++ common/argus_util.c
@@ -6634,7 +6634,6 @@ ArgusGetIndicatorString (struct ArgusParserStruct *par
 
 
 char argus_strbuf[MAXSTRLEN];
-u_short ArgusThisProto;
 
 void
 ArgusPrintSourceID (struct ArgusParserStruct *parser, char *buf, struct ArgusRecordStruct *argus, int len)
@@ -22679,8 +22678,8 @@ ArgusLog (int priority, char *fmt, ...)
             *tptr++ = buf[i];
          }
 
-         memset(buf, 0, MAXSTRLEN);
-         strncpy(buf, tbuf, MAXSTRLEN);
+         memset(buf, 0, sizeof(buf));
+         strncpy(buf, tbuf, sizeof(buf));
       }
  
       syslog (priority, "%s", buf);
